wrangler-docs/directives/set-column.md

```diff
@@ -62,6 +62,21 @@ would result in this record:
 
 ## Arithmetic and decimal operations example 
 
+Arithmetic operations can be used in several ways. 
+- To apply a simple arithmetic operation to a single, non-Decimal column, mathematical notation can be used, e.g. 
+```
+set-column :output wage1 * 2
+```
+- To apply an operation to a single column of type Decimal, use the `decimal` operations, e.g. 
+```
+set-column :decimal_op_1 decimal:add(wage_1,25)
+```
+- To apply operations to multiple columns (of any type), use the `arithmetic` operations, e.g. 
+```
+set-column :arithmetic_op_1 arithmetic:add(wage_1,wage_2)
+```
+For more information on working with numbers in Wrangler, consult [this Wiki page](https://cdap.atlassian.net/wiki/spaces/DOCS/pages/413466692/Working+with+numbers).
```

Comment:
For Google style, can we say "For more information about working with numbers in Wrangler, see..."<<just add the text/link>>